File: apollo/src/main/java/io/muun/apollo/domain/errors/IncorrectPasswordError.java
package io.muun.apollo.domain.errors;

import io.muun.apollo.external.UserFacingErrorMessages;

public class IncorrectPasswordError extends UserFacingError {

    public IncorrectPasswordError() {
        super(UserFacingErrorMessages.INSTANCE.incorrectPassword());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/passwd/IncorrectPasswordError.kt
package io.muun.apollo.domain.errors.passwd

import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.UserFacingError

class IncorrectPasswordError : UserFacingError(UserFacingErrorMessages.INSTANCE.incorrectPassword())
